libs/designer/src/lib/ui/mcp/parameters/ParameterEditor.tsx

```diff
@@ -127,16 +140,95 @@ export const ParameterEditor = ({
     },
   });
 
+  const onCastParameter = (value: ValueSegment[]): string => {
+    return (
+      parameterValueToString(
+        {
+          ...parameter,
+          value,
+        } as ParameterInfo,
+        /* isDefinitionValue */ false
```

Comment:
This need to be set to true.... It is a definition value and not used for dynamic calls